src/audio/dai.c

```diff
@@ -651,11 +651,11 @@ static int dai_comp_trigger_internal(struct comp_dev *dev, int cmd)
 
 		/* only start the DAI if we are not XRUN handling */
```

Comment:
@iuliana-prodan this is also the order ALSA uses to enable DAI and DMA, so in principle I agree with this change. Lets also hear Intel folks opinion on this as I remember they did the reverse change.

Reply:
The order is not my opinion, this is what is written in i.MX reference manual (one can check for "Suspend/resume a DMA channel with active hardware service requests").